cmd/capd-manager/main.go

```diff
@@ -35,6 +36,9 @@ import (
 )
 
 func main() {
+	flag.Set("v", "0")
+	flag.Parse()
+
 	cfg, err := config.GetConfig()
 	if err != nil {
 		panic(err)
```

Comment:
klog.Fatal?

Reply:
Yep, This is covered in https://github.com/kubernetes-sigs/cluster-api-provider-docker/issues/8

I'll time that out soon depending on if the contributor contributes or not